pre-backup

```diff
@@ -137,39 +137,25 @@ extract_versioned_apis() {
 
 validate_kinds() {
   local -a expected=(
-    buildconfigs
-    builds
     configmaps
-    cronjobs
     daemonsets
-    deploymentconfigs
     deployments
     endpoints
-    groups
     horizontalpodautoscalers
-    imagestreams
-    imagestreamtags
     ingresses
     jobs
     limitranges
     namespaces
     nodes
-    oauthaccesstokens
-    oauthclients
     persistentvolumeclaims
     persistentvolumes
-    projects
     replicasets
-    replicationcontrollers
     resourcequotas
-    rolebindings
```

Comment:
Why no `rolebindings`?

Reply:
Same as on comment related to `cronjobs`.